man/systemd-networkd-wait-online.service.xml

```diff
@@ -91,6 +91,7 @@
   <refsect1>
     <title>See Also</title>
     <para>
+      <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
```

Comment:
I prefer that this is located after systemd(1).

Reply:
No problem at all.